server/requesters/requesterCtrl.js

```diff
@@ -136,10 +136,10 @@ module.exports = {
       });
 
       var yelp = Yelp.createClient({
-        consumer_key: process.env.YELP_CONSUMER_KEY || Auth.yelp.consumer_key,
-        consumer_secret: process.env.YELP_CONSUMER_SECRET || Auth.yelp.consumer_secret,
-        token: process.env.YELP_TOKEN || Auth.yelp.token,
-        token_secret: process.env.YELP_TOKEN_SECRET || Auth.yelp.token_secret
+        consumer_key: process.env.YELP_CONSUMER_KEY || api_keys.consumer_key,
+        consumer_secret: process.env.YELP_CONSUMER_SECRET || api_keys.consumer_secret,
+        token: process.env.YELP_TOKEN || api_keys.token,
+        token_secret: process.env.YELP_TOKEN_SECRET || api_keys.token_secret
       });
```

Comment:
Could you move the keys outside of the exported function? Or do you feel its more appropriate where it is currently?